bucket/sunshine.json

```diff
@@ -13,12 +13,14 @@
             "hash": "c12904346cdf36a0848b35050f4ce6b45f5dde4c65191f0f00d9c1000d6a1759"
         }
     },
+    "pre_install": "Set-Content -Path \"$dir\\sunshine.bat\" -Value (@('@echo off', 'pushd %~dp0 && sunshine.exe && popd') -join \"`r`n\")",
     "bin": [
-        "sunshine.exe",
+        "sunshine.bat",
```

Comment:
Does this file need to exist in the zip? If so, there is no `sunshine.bat`, only `sunshine.exe`